libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -190,13 +338,17 @@ def get_ips(pool_file="resources/pool.json"):
     with open(pool_file) as f:
         pool_dict = json.loads(f.read())
         ips = pool_dict["ips"]
+        ip_to_node_type = []
+
+        if "ip_to_node_type" in pool_dict.keys():
```

Comment:
You can use also check using if `"ip_to_node_type" in pool_dict:` but whatever you think is best.